wpilibcExamples/src/main/cpp/examples/SelectCommand/cpp/RobotContainer.cpp

```diff
@@ -15,7 +23,7 @@ void RobotContainer::ConfigureButtonBindings() {
   // Configure your button bindings here
 }
 
-frc2::Command* RobotContainer::GetAutonomousCommand() {
+frc2::CommandPtr RobotContainer::GetAutonomousCommand() {
   // Run the select command in autonomous
-  return m_exampleSelectCommand.get();
+  return std::move(m_exampleSelectCommand);
```

Comment:
This will leave the field in an undefined state. Either return `.get()` (as a `Command*`, like before), or remove the field and return a new instance in the method. I recommend the former.